style.css

```diff
@@ -0,0 +1,81 @@
+ul.header {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
```

Comment:
- [ ] Please use `RGB` for color

Reply:
- [ ] It will be great to use an explanation for each requested change.
- [ ] Remember that the comment should be a suggestion instead of making it look like an order.

Reply:
- [ ] 🤜 Great for using `background-color` for your page. I suggest you to use **RGB** instead of **hex** color representation because it will rise **CSS** linter errors.